src/appleseed.studio/mainwindow/project/entityvalueprovider.h

```diff
@@ -0,0 +1,47 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
+// Copyright (c) 2014-2017 Francois Beaune, The appleseedhq Organization
```

Comment:
This is a new file, so you should replace these two copyright lines by a single one:

    // Copyright (c) 2017 Andrei Ivashchenko, The appleseedhq Organization